### DISCUSSION TITLE
Selecting and moving multiple stokes at a time

### DISCUSSION THREAD

**Discussion by YouFoundJK** [pubpub-zz/ppInk#117] [Q&A] [answered]
Is it possible move multiple stokes at a time, say I want to move everything within some rectangular region, is that possible?

Also it would be supercool if a freeform selection tool can be implemented. 

By the way, this is a supercool upgrade to gInk. Thanks Guys!

**Comment by pubpub-zz** [comment, marked as answer]
this feature will be release in the next days. a beta version is available in #97.
If you like it, do not hesitate to star it
